core/src/test/java/greencity/ModelUtils.java

```diff
@@ -43,25 +41,41 @@ public static EcoNewsAuthorDto getEcoNewsAuthorDto() {
     }
 
     public static Language getLanguage() {
-        return new Language(1L, "en", Collections.emptyList(), Collections.emptyList(),
-            Collections.emptyList());
+        return new Language(1L, "en", Collections.emptyList(), Collections.emptyList(), Collections.emptyList());
     }
 
     public static EcoNews getEcoNews() {
-        return new EcoNews(1L, ZonedDateTime.now(), TestConst.SITE, getUser(),
-            "title", "text", Collections.singletonList(getTag()));
+        return new EcoNews(1L, ZonedDateTime.now(), TestConst.SITE, getUser(), "title", "text",
+            Collections.singletonList(getTag()));
     }
 
     public static AddEcoNewsDtoRequest getAddEcoNewsDtoRequest() {
-        return new AddEcoNewsDtoRequest("title", "text",
-            Collections.singletonList("tag"));
+        return new AddEcoNewsDtoRequest("title", "text", Collections.singletonList("tag"));
     }
 
     public static AddEcoNewsDtoResponse getAddEcoNewsDtoResponse() {
-        return new AddEcoNewsDtoResponse(1L, getEcoNews().getTitle(),
-            getEcoNews().getText(), getEcoNewsAuthorDto(),
-            getEcoNews().getCreationDate(), TestConst.SITE,
-            Collections.singletonList("tag"));
+        return new AddEcoNewsDtoResponse(1L, getEcoNews().getTitle(), getEcoNews().getText(), getEcoNewsAuthorDto(),
```

Comment:
Don't use other methods tocreate dto or models - like getEcoNews(), because if this method will be changed  - it may cause tests crash that are using your getAddEcoNewsDtoResponse() method. Just put some random data.

Reply:
fixed, changes added to a newly created pull request #585